docs/guides/commands/samples/command_handler.cs

```diff
@@ -8,36 +8,36 @@
 
 public class Program
 {
-    private CommandService commands;
-    private DiscordSocketClient client;
-    private IServiceProvider services;
+    private CommandService _commands;
+    private DiscordSocketClient _client;
+    private IServiceProvider _services;
 
     static void Main(string[] args) => new Program().Start().GetAwaiter().GetResult();
```

Comment:
`static async Task Main(string[] args) => await new Program().StartAsync()`

Let's break out the C#7.1 features.